sdks/python/apache_beam/transforms/sql.py

```diff
@@ -31,8 +31,15 @@
 
 __all__ = ['SqlTransform']
 
+_SQL_PLANNERS = {
+    'zetasql': 'org.apache.beam.sdk.extensions.sql.zetasql.ZetaSQLQueryPlanner',
```

Comment:
I wonder if it would make more sense for this mapping to live on the Java side, and to have this parameter by called "dialect" or similar.

Reply:
Yeah I think that makes sense. Pushed a commit that moves it into the Java side. The only caveat is I had to move the external transform code out of SqlTransform itself (and into the expansion-service package) so that we can reference ZetaSQL.